src/wast-parser.cc

```diff
@@ -1335,6 +1349,19 @@ Result WastParser::ParseModuleFieldList(Module* module) {
       CHECK_RESULT(Synchronize(IsModuleField));
     }
   }
+
+  Result result = Result::Ok;
+
+  for (auto it : resolve_types_) {
+    result |= ResolveRefTypes(module, *it.target_types, it.vars, errors_);
```

Comment:
Why not `CHECK_RESULT` here?   I suppose so that we report more than just the first error?   Is that necessary though?  Do we do it elsewhere?

Reply:
I usually get multiple errors. But if you prefer a single error, I can change the code.